jest.config.ts

```diff
@@ -15,7 +15,10 @@ const config: Config.InitialOptions = {
 		'!**/*.d.ts',
 		'!**/node_modules/**',
 	],
-	testPathIgnorePatterns: ['<rootDir>/src/__tests__/e2e'],
+	testPathIgnorePatterns: [
+		'<rootDir>/src/__tests__/e2e',
+		'rehype-sanitize.test.ts',
```

Comment:
The updated version of `rehype-sanitize` is ESM only... and I couldn't seem to find a way to get Jest to work with it. There probably is a way, would love any input here.

See also [Slack thread](https://hashicorp.slack.com/archives/C02T80JTNS2/p1682028713094729) for potential alternatives.

Reply:
Updated to use a "dynamic config" like in `web-platform-packages` 👍